core/commands/repo.go

```diff
@@ -19,10 +21,17 @@ var RepoCmd = &cmds.Command{
 	},
 
 	Subcommands: map[string]*cmds.Command{
-		"gc": repoGcCmd,
+		"gc":   repoGcCmd,
+		"stat": repoStatCmd,
 	},
 }
 
+type RepoStat struct {
+	localPath string
+	szBytes   uint64
+	szBlocks  uint64
```

Comment:
+1 for the naming consistency with`NodeStat`. Though for size, this is how it is described in e.g. `os.FileInfo` https://golang.org/src/os/types.go?s=391:752#L6.